### PR TITLE
Telegram Bot Init

### PR DESCRIPTION
**Pull request by Drapegnik**
http://t.me/lzrsquashbot

### CODE REVIEW

File: bot/settings.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import os
+
+from dotenv import load_dotenv
+load_dotenv()
+
+token = os.getenv('TG_TOKEN')
```
Comment (UladBohdan) [inline bot/settings.py:6]: can we replace env variable with command-line argument? I hate env vars

Comment (UladBohdan) [reply bot/settings.py:6]: and you should hate them too

Comment (Drapegnik) [reply bot/settings.py:6]: imho cli args for configuration (params), but secrets should be in files (`.env` files)

File: bot/settings.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+groups = [-325409771]
+admins = ['drapegnik']
+
+GAME_FORMAT = '/game @Drapegnik 5-0 @uladbohdan'
```
Comment (KlicOgogo) [inline bot/settings.py]: `s1:s2` score format is better

**Review by KlicOgogo — APPROVED** [review]

**Pull request merged**